Locations.py

```diff
@@ -128,3 +131,106 @@ class MetroidPrimeLocation(Location):
     **mines_location_table,
     **magmoor_location_table
 }
+
+PICKUP_LOCATIONS: list[(int, int)] = [

```

Comment:
Nice, I was thinking we'd combine this with the locations list so essentially each location would have a name, AP id and then an associated memory relay identifier. I'm fine to keep it like this for now though 👍

Reply:
the problem is that it needs to be ordered the same as the patcher. is the location table using the same order?

Reply:
Ah I didn't realize that, it is likely not. I'll add a comment on that later so that is remembered 👍